packages/browser/src/plugins/segmentio/__tests__/retries.test.ts

```diff
@@ -131,3 +134,56 @@ describe('Batches retry 500s and 429', () => {
     expect(fetch.mock.lastCall[1].body).toContain('"retryCount":2')
   })
 })
+
+describe('retryQueue', () => {
+  let options: SegmentioSettings
+  let analytics: Analytics
+  let segment: Plugin
+  beforeEach(async () => {
+    jest.useRealTimers()
```

Comment:
I generally just just make this at afterEach -- and beforeEach can just have 'jest.useFakeTimers()`